### ISSUE TITLE
Can't seem to figure out how to import tags across platforms

### ISSUE THREAD

**Issue by CrawlinChaos** [tagspaces/tagspaces#249]
So, I have two computers I work off of: a Macbook Pro that I take with me and a Windows desktop computer that stays at home. What I would like to be able to do is export the tags from one computer and import them on the other, so I can keep my overall tag library updated between computers. However, when I export the JSON from TagSpaces on my Windows and then try to import it on my Mac, the 'Import Tags from File' window comes up, but when I confirm the import, nothing happens. I've tried multiple things to fix this, including changing the TagSpaces location to the directory with the JSON, removing the tag from the JSON file so it just reads 'tsm.json' and restarting TagSpaces after each attempt to import. To make things even weirder, the 'Import Tags from File' window only comes up when I try to import the first time. Every subsequent time it does not appear until I restart TagSpaces. Maybe I'm just don't know what the trick is to get importing to work, but even if I did, importing tags has to be made much easier than this. I might even suggest making an option to access the tag library of TagSpaces directly, if possible, or being able to create a dynamically updating JSON so that people can use Cloud-based services to sync the library between computers without having to import each time to update the tag library.

**Comment by uggrock** [comment]
The current functionality only adds new taggroups which does not currently exist. It does not sync or overwrite existing taggroups, which is actually a bug and should be fixed in the next release.

**Comment by CrawlinChaos** [comment]
Ah, I understand. So, the work around for this bug is to delete Tag Groups that you want to import which already exist. For example, if I want to import a JSON with a new Common Tags group, I need to delete the current Common Tags group and then perform the import. I tested this method and it does work. I'm glad that this is a bug and it will be fixed, because being able to import or sync tags easily is very important in my opinion. It not only opens up the possibility of cross platform work or work on multiple computers, but also the option for people to share tags with each other. You might even want to consider having an option to export individual Tag Groups after you fix the overwriting issue.

**Comment by Smadoski** [comment]
Has this issue ever been resolved?

im sharing the tsm.json file in .ts on OS X. I have an automatic process running which will capture the exported file from machine (a) rename it and copy to the .ts folder. In order to import the tags in to machine (b) I have to first delete the existing tag group. It then works. I have noticed ithat if you select two different file locations and add .ts folders with different .json files in each then just switching between location updates the tags immediately without having to delete first. Would it be possible to add a save tags to .ts on exit at some point?

As a side note I have been experimenting with Cryptomator and find that trying to sync tags between machines is almost unworkable. I have tried this on Dropbox, Box and iCloud with the same results and believe that the issue probably lies with the webdav component in Cryptomator not syncing the hidden .ts folder. As I'm writing I'm wondering why I didn't just try unhiding the folder? I will try this later.
